var/spack/repos/builtin/packages/gaussian/package.py

```diff
@@ -14,46 +15,261 @@ class Gaussian(Package):
+                'bsd/testl1',
+                'bsd/co1',
+                'bsd/txzip',
+                'bsd/comp',
+                'bsd/cachesize',
+                'bsd/g16test',
+                'bsd/ckall',
+                'bsd/setup-linda',
+                'bsd/save-working',
+                'bsd/fixlib',
+                'bsd/sumcpu',
+                'bsd/checkf',
+                'bsd/ren',
+                'bsd/make-ldir',
+                'bsd/updatelink',
+                'bsd/mkunixcd',
+                'bsd/updatelink1',
+                'bsd/gau-get',
+                'bsd/clone-working',
+                'bsd/make-edir',
+                'bsd/get-ref',
+                'bsd/gau-ranlib',
+                'bsd/ckwork',
+                'bsd/gau-unlimit',
+                'bsd/getx',
+                'bsd/tbzip',
+                'bsd/ssruny',
+                'bsd/extract-ghelp-16',
+                'bsd/tgzip',
+                'bsd/linda-ln',
+                'bsd/test-times',
+                'bsd/gau-hname',
+                'bsd/lxi-working',
+                'bsd/gzt',
+                'bsd/lx-working',
+                'bsd/x86type',
+                'bsd/compall',
+                'bsd/movemerged',
+                'bsd/linda-lnx',
+                'bsd/gau-fixexe',
+                'bsd/diff-src',
+                'bsd/findnew',
+                'bsd/fixlinda',
+                'bsd/rdmat-ext',
+                'tests/sd',
+                'tests/testfc',
+                'tests/cknan',
+                'tests/check-of',
+                'tests/runall',
+                'tests/sd2',
+                'tests/test-newzmat',
+                'tests/run-smallc',
+                'tests/run-smallal',
+                'tests/d1',
+                'tests/search.csh',
+                'tests/diff.csh',
+                'tests/run-smallb',
+                'tests/run-scrf',
+                'tests/sd1',
+                'tests/run-quart',
+                'tests/runcube',
+                'tests/run-smallb1',
+                'tests/diff-mfl',
+                'tests/runpi',
+                'tests/dcube',
+                'tests/run-bigal',
+                'tests/run-bigb',
+                'tests/tgaunbo7',
+                'tests/run-cis',
+                'tests/run-bigbl',
+                'tests/run-bigc',
+                'tests/test-gen',
+                'tests/run22',
+                'tests/run-smalla',
+                'tests/sdt',
+                'tests/run-small',
+                'tests/run6',
+                'tests/run-big',
+                'tests/sd3',
+                'tests/searchlog.csh',
+                'tests/run-biga',
+                'tests/tgaunbo6',
+                'tests/submit.csh',
+                'tests/run-dft',
+                'tests/run-smallbl',
+                string=True,
+                backup=False)
+
+            filter_file('/bin/csh', spec['tcsh'].prefix.bin.tcsh,
```

Comment:
The makefiles need an absolute path to `TCSH`, that's why you're not using `/usr/bin/env tcsh` as in the filters above?

Reply:
No, the reason why I use the full path here is because `make` doesn't have a problem with long path names (unlike shebangs in shell scripts)